Userland/Applications/Presenter/SlideObject.h

```diff
@@ -31,19 +31,22 @@ class SlideObject : public Core::Object {
 
     static ErrorOr<NonnullRefPtr<SlideObject>> parse_slide_object(JsonObject const& slide_object_json, NonnullRefPtr<GUI::Window> window);
 
-    // FIXME: Actually determine this from the file data.
-    bool is_visible_during_frame([[maybe_unused]] unsigned frame_number) const { return true; }
+    bool is_visible_during_frame([[maybe_unused]] unsigned frame_number) const { return m_frames.contains(frame_number) || m_frames.is_empty(); }
```

Comment:
Putting the `m_frames.is_empty()` check first will save having to do a hash lookup.